site2/tools/pulsar-config-doc-gen.sh

```diff
@@ -52,10 +52,3 @@ $JAVA -cp "$CLIENT_CP:$(cat "$f")" $GEN_DOCS_CLIENT -c $CLIENT_CONF.ClientConfig
 $JAVA -cp "$CLIENT_CP:$(cat "$f")" $GEN_DOCS_CLIENT -c $CLIENT_CONF.ProducerConfigurationData > "$CLIENT_DIR"/client-configuration-producer.md
 $JAVA -cp "$CLIENT_CP:$(cat "$f")" $GEN_DOCS_CLIENT -c $CLIENT_CONF.ConsumerConfigurationData > "$CLIENT_DIR"/client-configuration-consumer.md
 $JAVA -cp "$CLIENT_CP:$(cat "$f")" $GEN_DOCS_CLIENT -c $CLIENT_CONF.ReaderConfigurationData > "$CLIENT_DIR"/client-configuration-reader.md
-
-# copy CLI tools docs
-tools="bookkeeper pulsar-daemon pulsar-shell"
-for tool in $tools
-do
-    cp "$WEBSITE/docs/reference-cli-$tool.md" "$VERSIONED_DIR/$tool/$tool.md"
-done
```

Comment:
This code snippet along with the existence of the `reference-cli-*` files causes the current https://pulsar.apache.org/reference/#/next/bookkeeper/bookkeeper page still have a wrongly parsed yaml header.

We need to remove all this stuff to finish what @Anonymitaet proposed at https://github.com/apache/pulsar/pull/18101

Reply:
@SignorMercurio would you pls help to double-check it? thanks.

Reply:
Yes, this code snippet is no longer needed and should be removed. So does `reference-cli-*.md`. Thanks @tisonkun!

Reply:
OK, thank you all!